spec/models/order_spec.rb

```diff
@@ -34,59 +42,150 @@
           expect(order.errors.keys).to include(:status)
         end
       end
+    end
 
-      # data validations
-      it "validates :status, presence: true, format: { with: VALID_STATUS_REGEX }" do
-        # TODO
-      end
+    context "buyer_info" do
+      it "does not require any buyer info for pending orders" do
+        order = Order.create
 
-      it "validates_presence_of :buyer_email, unless: :pending?" do
-        # TODO
-      end
+        buyer_fields = [:buyer_name, :buyer_email, :buyer_address, :buyer_card_short, :buyer_card_expiration]
 
-      it "validates_format_of :buyer_email, with: VALID_EMAIL_REGEX, unless: :pending?" do
-        # TODO
+        buyer_fields.each do |field|
+          expect(order.errors.keys).to_not include(field)
```

Comment:
It sounds like `to_not` and `not_to` are aliases and the only reason to favor one of the other is personal preference and readability. In this case, I think we still have enough to do that it's not worth going through and fixing all the specs to match, but I will definitely keep this in mind if we have extra polishing time. :) ([source](http://comments.gmane.org/gmane.comp.lang.ruby.rspec.user/19972)).